frontend/public/components/utils/drag-drop-context.tsx

```diff
@@ -1,6 +1,15 @@
-import { DragDropContext } from 'react-dnd';
+import * as React from 'react';
+import { DndProvider } from 'react-dnd';
 import HTML5Backend from 'react-dnd-html5-backend';
 
-// Need to create this decorator, because of the issue with react-dnd module:
-// https://github.com/react-dnd/react-dnd/issues/186#issuecomment-282789420
-export default DragDropContext(HTML5Backend);
+const withDragDropContext = <TProps extends {}>(
+  Component: React.ComponentClass<TProps> | React.StatelessComponent<TProps>,
```

Comment:
```suggestion
  Component: React.ComponentType<TProps>,
```

Reply:
you should specify TProps for every `withDragDropContext` usage. For example in `droppable-edit-yaml.tsx` you should have `export const DroppableEditYAML = withDragDropContext<DroppableEditYAMLProps>(...`